README.md

```diff
@@ -1,23 +1,15 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Hello World
 
-# Project Name
-
-> One paragraph statement about the project.
+> Repo where I practice the rules and correct behavior when 'Creating Repos' comes to action. 
```

Comment:
- [ ] Kindly edit this line to provide a brief summary of what the project is about.

Reply:
Ok, got it @Ghiftee, It is done on commit 2c4e8e2304c54fe7a22b8dc6257f5ae3b7521a0e